### PR TITLE
Use sparkle assistant preview gallery + add authors

### PR DESCRIPTION
**Pull request by flvndvd**
This PR addresses the remaining issue outlined in https://github.com/dust-tt/dust/issues/2912. It incorporates the `AssistantPreview` component, a recent addition to our Sparkle collection (https://github.com/dust-tt/dust/pull/3049). This update includes fetching and displaying author(s) information in the assistant gallery view, aligning with the designs specified in the [Figma file](https://www.figma.com/file/QOxHwppG64GtPocpDhS6Y7/Product?node-id=2914%3A9312&mode=dev) and the details discussed in the [Slack thread](https://dust4ai.slack.com/archives/C050A0S2Z7F/p1702910140159249).

**Authors**

image

### CODE REVIEW

**Review by Duncid — APPROVED** [review]

File: front/pages/w/[wId]/assistant/gallery.tsx [hunk 1]
```diff
@@ -12,7 +12,7 @@ import { useRouter } from "next/router";
 import { useState } from "react";
 
 import { AssistantDetails } from "@app/components/assistant/AssistantDetails";
-import { AssistantPreview } from "@app/components/assistant/AssistantPreview";
```
Comment (spolu) [inline front/pages/w/[wId]/assistant/gallery.tsx:15]: Shouldn't the assiociated be removed since it now comes from sparkle?

**Review by spolu — APPROVED** [review]
LGTM!

File: front/pages/w/[wId]/assistant/gallery.tsx (hunk 1 above)
Comment (flvndvd) [reply front/pages/w/[wId]/assistant/gallery.tsx:15]: It's still used in the empty new conversation screen. I'll remove it once I implemented the associated variant.

Comment (spolu) [reply front/pages/w/[wId]/assistant/gallery.tsx:15]: Roger that :+1:

**Pull request merged**